src/modules/fancyzones/FancyZonesTests/UnitTests/AppZoneHistoryTests.Spec.cpp

```diff
@@ -335,9 +335,8 @@ namespace FancyZonesUnitTests
                 .monitorId = { .deviceId = { .id = L"DELA026", .instanceId = L"5&10a58c63&0&UID16777488" } },
                 .virtualDesktopId = FancyZonesUtils::GuidFromString(L"{39B25DD2-130D-4B5D-8851-4791D66B1539}").value()
             };
-            const auto window = Mocks::WindowCreate(m_hInst);
```

Comment:
This might actually be needed `Mocks::WindowCreate` is doing something, even if the resulting returning value is not used. @SeraphimaZykova , correct me if I'm wrong, but Mocks::WindowCreate likely still needs to be called.

Reply:
It's safe to delete this line, it's won't affect the test. This line was forgotten there I guess.